lib/core/src/server/utils/load-custom-babel-config.js

```diff
@@ -16,19 +16,37 @@ function removeReactHmre(presets) {
 // Tries to load a .babelrc and returns the parsed object if successful
 function loadFromPath(babelConfigPath) {
   let config;
+  const error = {};
 
   if (fs.existsSync(babelConfigPath)) {
     const content = fs.readFileSync(babelConfigPath, 'utf-8');
+
+    try {
+      // eslint-disable-next-line global-require, import/no-dynamic-require
+      config = require(babelConfigPath);
+      logger.info('=> Loading custom babel config as JS');
+    } catch (e) {
+      error.js = e;
+    }
     try {
-      config = /^module.exports/.test(content)
-        ? require(babelConfigPath) // eslint-disable-line
-        : JSON5.parse(content);
-      config.babelrc = false;
+      config = JSON5.parse(content);
       logger.info('=> Loading custom babel config');
     } catch (e) {
-      logger.error(`=> Error parsing babel config file: ${e.message}`);
-      throw e;
+      error.json = e;
     }
+
+    if (!config) {
+      logger.error(`=> Error parsing babel config file: ${babelConfigPath}
+      
+      We tried both loading as JS & JSON, neither worked.
+      Maybe there's a syntax error in the file?`);
+      logger.error(`=> From JS loading we got: ${error.js.message}`);
+      logger.error(`=> From JSON loading we got: ${error.js.message}`);
```

Comment:
This line should say `error.json.message`, shouldn’t it?

Reply:
good catch!

Reply:
Sorry, I’m afraid not :(